javascript/apps/taiga/src/app/modules/auth/feature-sign-up/components/signup/signup.component.html

```diff
@@ -29,14 +29,19 @@
         </tg-ui-error>
       </ng-container>
     </tg-ui-input>
-    <tg-ui-inline-notification
-      aria-live="polite"
+
+    <tg-ui-context-notification
```

Comment:
The size of the 'context notification' still not the same as the design. Have we created the issue as it's requested in the design review?